lib/linguist/languages.yml

```diff
@@ -3752,6 +3752,19 @@ RMarkdown:
   - ".rmd"
   tm_scope: source.gfm
   language_id: 313
+RPC:
+  type: programming
+  aliases:
+  - rpcgen
+  - oncrpc
+  - xdr
+  ace_mode: c_cpp
+  extensions:
+  - ".x"
+  interpreters:
+  - rpcgen
+  tm_scope: none
```

Comment:
> *Vim has syntax highlighting for rpcgen but I haven't found any other syntax highlighting. I can write one if necessary.*

You could use Objective-C's grammar for highlighting, as the language appears to be similar enough in syntax. Otherwise, a `tm_scope: none` will cause all RPC files to be shown on GitHub without colour.